OpenRA.Mods.Common/Orders/PowerDownOrderGenerator.cs

```diff
@@ -0,0 +1,205 @@
+﻿using System.Collections.Generic;
```

Comment:
This file is missing the header.

Reply:
ok

Reply:
Strange here,  
![note](https://user-images.githubusercontent.com/13763394/71541355-8154be00-2992-11ea-85b8-29fa3c892996.jpg)
I have already add this head in my code.

Reply:
During your test it is missing or something?

Reply:
No, I mean the copyright header.

Reply:
Oh! I don't notice that. I will fix it.